### ISSUE TITLE
Can't reimplement the final result

### ISSUE THREAD

**Issue by bobwan1995** [doc-doc/vRGV#6]
I can't reimplement the final result (only 20, 14, 7 for s/o/all with threshold 0.5) as your paper reported. I convert videos to images with 
> ffmpeg -i xxx.mp4 -r video_fps -q:v 2 xxx/%06d.jpeg

Is there anything different from yours? 

I have 2 more questions:
1. Why not shuffle train set?
2. You'd better mention that the evaluation code can only work when batchsize=1 at the inferent stage.

**Comment by doc-doc** [comment]
Hi, thanks for the comments. 
For your first question, pls set it to True for shuffling on training data. (I cannot quit remember why i set it to false because i did lot of analysis after the model was trained) 
For evaluation, i guess you are saying the attention extraction part and that's yes, because i saved the values per video online. It can be easily fixed by saving the video_name& framed id along with the values and merging into videos offline.

For your results, i am not sure which parts are the problems, and i do not think it because of shuffling. I will release our model later.

**Comment by bobwan1995** [comment]
Actually I've downloaded your model parameters and loaded it to evaluate. The results (24, 20, 13 for s/o/all with threshold 0.5) are better than I trained but is still much lower than the paper reported. I guess it might be caused by the video2image conversion, is that possible? Thanks a lot!

**Comment by bobwan1995** [comment]
I'm sure I followed the instruction of this repo and didn't make any modifications but get a confusing result.  I really want to do some research based on this work, I will be most grateful if you could provide me a detailed tutorial that can get the result as your paper mentioned (including data processing), maybe when you have more spare time (I guess after cvpr). Thank you so much!

**Comment by doc-doc** [comment]
Ok, I will detail the data processing part later. Before that, please first check 1) whether your number of extracted frames (for each video) is same as the annotated frame_count in the annotation file. Possibly you can visualize the annotations with your extracted frames, then 2) check your sampling: sample 512 frames for each video first, and further sample 120 from the 512 frames, bothing using linspace().

**Comment by bobwan1995** [comment]
Sorry, I found I directly sampled 120 frames for each video (not 512 to 120). I will visualize the annotations with my extracted frames. But you never told me whether the command I use to convert video to images is correct (video_fps is the annotated frame_count for each video in the annotation file):

>ffmpeg -i xxx.mp4 -r video_fps -q:v 2 xxx/%06d.jpeg

I found the index of extracted images with this command starts from 1, not 0 in your codes. Can you tell me the conversion command you use? Thank you so much!

**Comment by doc-doc** [comment]
I see, pls try
```
ffmpeg -i vname.mp4 -start_number 0 ./%06d.JPEG
```
If this does not work for you and the index starts from 1, you can rename the frames.

**Comment by rookiecm** [comment]
> I see, pls try
> 
> ```
> ffmpeg -i vname.mp4 -start_number 0 ./%06d.JPEG
> ```
> 
> If this does not work for you and the index starts from 1, you can rename the frames.

Hi, I have the same problem. Could you please share your extracted features?
